service/src/integrationTest/kotlin/fi/espoo/evaka/pis/PersonIntegrationTest.kt

```diff
@@ -62,6 +62,61 @@ class PersonIntegrationTest : PureJdbiTest() {
         Assertions.assertTrue(personHasMessageAccount(personId))
     }
 
+    @Test
+    fun `getPersonReferences returns references to person and child tables excluding self-references and message_account`() {
+        /*
+        This should exclude tables where data is always deleted with person and not transferred:
+        - child
+        - guardian
+        - message_account
+         */
+        val references = db.read { it.getTransferablePersonReferences() }
+        Assertions.assertEquals(38, references.size)
+        Assertions.assertEquals(
+            listOf(
+                PersonReference("absence", "child_id"),
+                PersonReference("application", "child_id"),
+                PersonReference("application", "guardian_id"),
+                PersonReference("application", "other_guardian_id"),
+                PersonReference("assistance_action", "child_id"),
+                PersonReference("assistance_need", "child_id"),
+                PersonReference("attachment", "uploaded_by_person"),
+                PersonReference("backup_care", "child_id"),
+                PersonReference("backup_pickup", "child_id"),
+                PersonReference("bulletin_instance", "receiver_person_id"),
+                PersonReference("bulletin_receiver", "child_id"),
+                PersonReference("child_attendance", "child_id"),
+                PersonReference("daily_service_time", "child_id"),
+                PersonReference("daycare_daily_note", "child_id"),
+                PersonReference("family_contact", "child_id"),
+                PersonReference("family_contact", "contact_person_id"),
+                PersonReference("fee_alteration", "person_id"),
+                PersonReference("fee_decision", "head_of_family"),
+                PersonReference("fee_decision", "partner"),
+                PersonReference("fee_decision_part", "child"),
+                PersonReference("fridge_child", "child_id"),
+                PersonReference("fridge_child", "head_of_child"),
+                PersonReference("fridge_partner", "person_id"),
+                PersonReference("income", "person_id"),
+                PersonReference("invoice", "head_of_family"),
+                PersonReference("invoice_row", "child"),
+                PersonReference("koski_study_right", "child_id"),
+                PersonReference("messaging_blocklist", "blocked_recipient"),
+                PersonReference("messaging_blocklist", "child_id"),
+                PersonReference("new_fee_decision", "head_of_family_id"),
+                PersonReference("new_fee_decision", "partner_id"),
+                PersonReference("new_fee_decision_child", "child_id"),
+                PersonReference("placement", "child_id"),
+                PersonReference("service_need", "child_id"),
+                PersonReference("varda_child", "person_id"),
```

Comment:
should varda_child data be transferred or deleted?

Reply:
Probably just transferred.